README.md

```diff
@@ -35,7 +35,7 @@ MinIO provides multiple methods for accessing and managing the MinIO Tenant:
 The MinIO Console provides a graphical user interface (GUI) for interacting with
 MinIO Tenants. 
 
-IMAGE
+<img src=docs/images/console-dashboard.png width="600px"/>
```

Comment:
You should use markdown syntax here @ravindk89 using HTML is not advised generally

Reply:
Done